src/application/worker/index.worker.js

```diff
@@ -341,13 +341,9 @@ async function start() {
         }
       }
 
-      store.commit("groups/SWAP", {});
-      store.commit("modules/SWAP", {});
-      store.commit("inputs/SWAP", {});
-
-      store.commit("groups/CLEAR_SWAP", {});
-      store.commit("modules/CLEAR_SWAP", {});
-      store.commit("inputs/CLEAR_SWAP", {});
+      store.commit("groups/SWAP");
+      store.commit("modules/SWAP");
+      store.commit("inputs/SWAP");
 
```

Comment:
CLEAR_SWAP removed as we already clear the swap by setting the default state when swapped.